tests2/omit.test.ts

```diff
@@ -345,7 +345,10 @@ describe(`Omit (${dbProvider})`, () => {
   })
```

Comment:
It doesn't seem like we test the omitting/not behaviour around `isFilterable`/`isOrderable`, can we add that?

Reply:
this will be added in future work, before the next stable